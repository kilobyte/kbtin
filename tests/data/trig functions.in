#ali m {#math x $0;#show $0 â†’ $x}
m #angle 1 0
m #angle 1 1
m #angle 0 1
m #sinus 3*30
m #cosinus 90-0
m (450 - #angle 50 -50) % 360
m #sqrt 0.25
m (#sqrt 2)/0.0001
