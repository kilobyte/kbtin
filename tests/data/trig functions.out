#angle 1 0 → 0
#angle 1 1 → 45
#angle 0 1 → 90
#sinus 3*30 → 1
#cosinus 90-0 → 0
(450 - #angle 50 -50) % 360 → 135
#sqrt 0.25 → 0.5
(#sqrt 2)/0.0001 → 14142
